src/Application/Command/CreateMilestones.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Laminas\AutomaticReleases\Application\Command;
+
+use Laminas\AutomaticReleases\Git\Value\SemVerVersion;
+use Laminas\AutomaticReleases\Github\Api\GraphQL\Query\GetGithubMilestone;
+use Laminas\AutomaticReleases\Github\Api\V3\CreateMilestone;
+use Laminas\AutomaticReleases\Github\Api\V3\CreateMilestoneFailed;
+use Laminas\AutomaticReleases\Github\Event\Factory\LoadCurrentGithubEvent;
+use Laminas\AutomaticReleases\Github\Value\RepositoryName;
+use Psr\Log\LoggerInterface;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+final class CreateMilestones extends Command
+{
+    private LoadCurrentGithubEvent $loadEvent;
+    private GetGithubMilestone $getMilestone;
+    private CreateMilestone $createMilestone;
+    private LoggerInterface $logger;
+
+    public function __construct(
+        LoadCurrentGithubEvent $loadEvent,
+        GetGithubMilestone $getMilestone,
+        CreateMilestone $createMilestone,
+        LoggerInterface $logger
+    ) {
+        parent::__construct('laminas:automatic-releases:create-milestones');
+
+        $this->loadEvent       = $loadEvent;
+        $this->getMilestone    = $getMilestone;
+        $this->createMilestone = $createMilestone;
+        $this->logger          = $logger;
+    }
+
+    public function execute(InputInterface $input, OutputInterface $output): int
+    {
+        $milestoneClosedEvent = ($this->loadEvent)();
+        $repositoryName       = $milestoneClosedEvent->repository();
+        $releaseVersion       = $milestoneClosedEvent->version();
+        $milestone            = ($this->getMilestone)($repositoryName, $milestoneClosedEvent->milestoneNumber());
+
+        $milestone->assertAllIssuesAreClosed();
```

Comment:
Does this matter to us? Creating milestones is non-destructive, while creating releases has a downstream effect. Perhaps we can skip this check completely?

Reply:
Agreed with @Ocramius. Additionally, this will likely happen as a late step in the workflow, which means we'll be aborting the workflow if this is not already true.